app/forms/waste_carriers_engine/renewal_information_form.rb

```diff
@@ -6,8 +6,8 @@ class RenewalInformationForm < BaseForm
 
     def initialize(transient_registration)
       super
-      self.type_change = @transient_registration.registration_type_changed?
-      self.total_fee = @transient_registration.fee_including_possible_type_change
+      self.type_change = transient_registration.registration_type_changed?
```

Comment:
NIT Adding a line between super and this as per other changes

```suggestion
      
      self.type_change = transient_registration.registration_type_changed?
```